website/docs/sql_queries.md

```diff
@@ -93,6 +106,9 @@ see all changes in a given time window and not just the latest values.
 
 Please refer to [configurations](/docs/basic_configurations) section for the important configuration options.
 
+### Query Indexes and Timeline
```

Comment:
provide type values for different mdt partitions

use sub query within payload